estate/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    'name': 'Estate',
+    'version':'1.0.0',
+    'depends': ['base'],
+    'description': """ 
+        This is a base version of the Estate module for real estate management. 
+    """,
+    'license': 'LGPL-3',
+    'data': [
+        'security/ir.model.access.csv',
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False,
```

Comment:
```suggestion
    'application': True,
```
`installable` is True by default no need to add it explicitely
`auto_install` is only useful when you have other modules as dependencies, it makes no sense here.